src/less/navbar.less

```diff
@@ -771,6 +764,12 @@
       }
     }
   }
+  border-bottom: 1px solid @gray-100;
+}
+/////
+.navbar-default {
+  background-color: @navbar-default-bg;
+  border-bottom: 1px solid @gray-100;
```

Comment:
Can we remove the navbar-default border-bottom and replace with box-shadow: 0 2px 2px rgba(0,0,0,0.12);